packages/calcite-design-tokens/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@esri/calcite-design-tokens",
+  "version": "1.0.0",
+  "description": "Esri's Calcite Design System Tokens",
+  "homepage": "https://github.com/Esri/calcite-design-system",
+  "keywords": [
+    "Calcite",
+    "Calcite Design System",
+    "design tokens",
+    "tokens"
+  ],
+  "main": "./dist/js/calcite-headless.js",
+  "files": [
+    "./dist",
+    "./src",
+    "README.md"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Esri/calcite-design-system.git"
+  },
+  "author": {
+    "name": "Esri"
+  },
+  "license": "SEE LICENSE IN README.md",
+  "bugs": {
+    "url": "https://github.com/Esri/calcite-design-system/issues"
+  },
+  "scripts": {
+    "build:tokens": "node support/run.js",
+    "build:ts": "tsc --project tsconfig.prod.json",
+    "build": "npm run build:ts && npm run build:tokens",
+    "clean": "rimraf dist",
+    "lint:eslint": "eslint --ext .ts,.tsx --fix .",
+    "lint:markdown": "markdownlint --fix --disable MD024 MD013 MD041 MD033",
+    "lint:prettier": "prettier --write .",
+    "lint": "npm run lint:eslint && npm run lint:markdown && npm run lint:prettier",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@tokens-studio/sd-transforms": "0.8.2",
+    "change-case": "4.1.2",
+    "style-dictionary": "3.7.1"
```

Comment:
All of these dependencies were already added to the root `package.json` so they can be removed here.